cspell.json

```diff
@@ -201,6 +199,7 @@
     "typeahead",
     "typeof",
     "unauthorizedemptystate",
+    "useravatar",
```

Comment:
```suggestion
```

Let’s keep the cspell disables for `useravatar` inline since I think it only applies when we’re putting it in the URL. (I do realize a lot of our other components probably made it into cspell for the same reason, but it does make us a bit more prone to spelling/stylistic errors.)